components/common/Modal/FolderModal/FolderModal.tsx

```diff
@@ -0,0 +1,72 @@
+import { useMemo } from "react";
+import BaseModeal from "../BaseModal/BaseModal";
+import styles from "./folder.module.css";
+import { ADD_LINK, ADD_FOLDER, EDIT } from "constants/strings";
+import { BaseModalProps } from "../BaseModal/BaseModal";
+import type { Folder } from "hooks/useGetFolders";
+
+interface Props extends BaseModalProps {
+  link?: string;
+  deleted?: string;
+  list?: Folder[] | null;
```

Comment:
데이터에 대해서 자세히 표현되는게 좋을 것 같네요!
```suggestion
  folderList?: Folder[] | null;
```